ad-hoc/washington-facilities-percent-vax.R

```diff
@@ -0,0 +1,37 @@
+library(behindbarstools)
+library(ggplot2)
+library(tidyverse)
+
+# number of facilities reporting vax data in each state
+latest_scraped %>%
```

Comment:
Where is latest_scraped defined? Should there be a line above this, something like `latest_scraped <- read_scrape_data()`?

Reply:
Yep, left that out when I moved code to a new file!